File: CPP/Clothing.cpp
#include <string> // String Library
#include "Product.cpp"

using namespace std; // Standard namespace, glibc::std

class Clothing : public Product {
    private:
        // Attributes
        string size; // Clothing Size
        string material; // Clothing material
        char gender; // Clothing gender

    public:
        /* Constructors */ 

        // Empty constructor
        Clothing(){
            this->size = "n/a";
            this->material = "n/a";
            this->gender = '-';
        }
        
        // Non-empty constructor
        Clothing(string size, string material, char gender) {
            this->size = size;
            this->material = material;
            this->gender = gender;
        }
        
        /* Methods */ 
        
        // Get Methods
        string getSize() { return this->size; }
        string getMaterial() { return this->material; }
        char getGender() { return this->gender; }
        
        // Set Methods
        void setSize(string size) { this->size = size; }
        void setMaterial(string material) { this->material = material; }
        void setGender(char gender) { this->gender = gender; }
};

File: PHP/Clothing.php
<?php

class Clothing extends Product {
    // Private attributes
    private $size = '';
    private $material = '';
    private $gender = '';

    /* Constructor */ 
   public function __construct($size = 'n/a', $material = 'n/a', $gender = 'n/a') {
        $this->size = $size;
        $this->material = $material;
        $this->div_material = $div_material;
    }

    /* Methods */ 
    // Get methods
    public function getSize() { return $this->size; }
    public function getMaterial() { return $this->material; }
    public function getGender() { return $this->gender; }

    // Set methods
    public function setSize($size) { $this->size = size };
    public function setMaterial($material) { $this->material = material };
    public function setGender($gender) { $this->gender = gender };
}
